language/stdlib/modules/LibraAccount.move

```diff
@@ -267,6 +263,61 @@ module LibraAccount {
             }
         );
     }
+    spec fun deposit {
+        include DepositAbortsIf<Token>;
+        include DepositEnsures<Token>;
+    }
+    spec schema DepositAbortsIf<Token> {
+        payer: address;
+        payee: address;
+        to_deposit: Libra<Token>;
+        metadata_signature: vector<u8>;
+        aborts_if !spec_has_account_operations_cap();
+        aborts_if to_deposit.value == 0;
+        aborts_if !exists<LibraAccount>(payer);
+        aborts_if !exists<LibraAccount>(payee);
+        aborts_if !exists<Balance<Token>>(payee);
+        aborts_if global<Balance<Token>>(payee).coin.value + to_deposit.value > max_u64();
+        include TravelRuleAppliesAbortsIf<Token>;
+        aborts_if spec_travel_rule_applies<Token>(payer, payee, to_deposit.value)
+                        && len(metadata_signature) != 64;
```

Comment:
Might make sense to extract this to a `spec fun signature_correct(metadata, metadata_signature, payee, amount)` that (for now) just checks the length of `metadata_signature`, but documents the abort conditions that are missing (cryptographic signature verification fails or `metadata != payer | amount | domain_separator`)